eventkit_cloud/ui/static/ui/app/components/CreateDataPack/FootprintDisplay.tsx

```diff
@@ -0,0 +1,100 @@
+import React from "react";
+import {createStyles, withStyles, FormControlLabel, Switch} from "@material-ui/core";
+import * as PropTypes from "prop-types";
+import {Props} from "./FootprintDisplay";
+import Card from "@material-ui/core/Card";
+import ListItemText from "@material-ui/core/ListItemText";
+import Typography from "@material-ui/core/Typography";
+import theme from "../../styles/eventkit_theme";
+
+const jss = () => createStyles({
+    footprintBox: {
+        backgroundColor: theme.eventkit.colors.selected_primary,
+        marginLeft: '33px',
+        maxWidth: '550px',
+        minWidth: '175px',
+        outline: '2px solid #4598bf',
+        paddingLeft: '10px',
+        paddingTop: '5px',
+    },
+    noPadding: {
+        padding: '0',
+    },
+    listItemText: {
+        paddingLeft: '10px',
+    },
+    formControlLabel: {
+        paddingLeft: '10px',
+        fontSize: '12px',
+        paddingRight: '3px'
+    },
+    switchBase: {
+    color: theme.eventkit.colors.white,
+      '&$checked': {
+        color: theme.eventkit.colors.primary,
+    },
+  },
+    checked: {},
+});
+
+export interface Props {
+    classes: { [className: string]: string };
+}
+
+export interface State {
+    checked: boolean;
+}
+
+export class FootprintDisplay extends React.Component<Props, State> {
+    static contextTypes = {
+        config: PropTypes.object,
+    };
+
+    constructor(props: Props) {
+        super(props);
+        this.handleCheckClick = this.handleCheckClick.bind(this);
+
+        this.state = {
+            checked: false,
+        };
+    }
+
+    handleCheckClick = () => {
+        this.setState({ checked: !this.state.checked });
+    }
+
+    render() {
+        const {classes} = this.props;
+
+        return (
+            <Card
+                className={classes.footprintBox}
+            >
+                <div>
+                    {/* using ListItem for future functionality that will add to this list */}
+                    <ListItemText
+                        className={classes.listItemText}
+                        primary={
+                            <Typography style={{fontSize: '12px'}}>
```

Comment:
We should wrap multiline JSX like this with parentheses. 

As seen here: https://github.com/EventKit/eventkit-cloud/blob/master/eventkit_cloud/ui/static/ui/app/components/CreateDataPack/MapDrawer.tsx#L275